manifests/helm-profiles/platform-gke.yaml

```
@@ -1,2 +1,2 @@
 cni:
-  cniBinDir: /home/kubernetes/bin
+  cniBinDir: ""
```

```suggestion
  cniBinDir: "" # intentionally unset for gke to allow template-based autodetection to work
```

Nit for discoverability in the future. I'll approve and put a hold on the PR so you can accept this, remove the label, and let it merge, or ignore, remove the label, and merge - your call.